Marcio/Notifications-new-content (#2343)
cloudflare/cloudflare-docs: products/fundamentals/src/content/notifications/edit-notification.md

---
order: 9
pcx-content-type: how-to
---

# Edit a Notification

1. Sign in to your [Cloudflare dashboard](https://dash.cloudflare.com/login).
1. Click **Notifications** on the upper right corner.
1. Click **Edit** on the Notification you want to modify.
1. Make changes as needed and click **Save**.

> This will take you back to your list of Notifications.

Reviewer: ```suggestion
The browser will navigate back to the list of Notifications.
```
